### PR TITLE
[bitnami/rabbitmq-cluster-operator] feat: add revisionHistoryLimit

### PR DESCRIPTION
**Pull request by KyriosGN0**
### Description of the change

Added revisionHistoryLimit parameter for cluster-operator and message topology operator

### Benefits

Users will be able to set this parameters to control the number of replicasets to keep in k8s

### Possible drawbacks

None
### Applicable issues

None

### Additional information

None
### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by KyriosGN0** [comment]
@andresbono help would be greatly appreciated

File: bitnami/rabbitmq-cluster-operator/Chart.yaml [hunk 1]
```diff
@@ -37,4 +37,4 @@ maintainers:
 name: rabbitmq-cluster-operator
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq-cluster-operator
-version: 4.1.0
+version: 4.1.1
```
Comment (andresbono) [inline bitnami/rabbitmq-cluster-operator/Chart.yaml:40]: Let's bump the minor version as this is introducing new parameters:

```suggestion
version: 4.2.0
```

**Comment by andresbono** [comment]
@jotamartos, reviewing this one as this is a follow-up of https://github.com/bitnami/charts/pull/24497.

**Review by andresbono — APPROVED** [review]
LGTM! Thank you for your contribution!

**Pull request merged**